cl/phase1/network/services/aggregate_and_proof_service.go

```diff
@@ -179,6 +210,7 @@ func (a *aggregateAndProofServiceImpl) ProcessMessage(
 			aggregateAndProof.SignedAggregateAndProof.Message.Aggregate,
 			attestingIndicies,
 		)
+		a.seenAggreatorIndexes.Add(seenIndex, struct{}{})
```

Comment:
Shouldn't we clean this after some time?

Reply:
it is an lru cache it gets automatically cleaned already, after n elements